src/mme-app/handlers/attach_aia_ula.c

```diff
@@ -294,7 +294,7 @@ post_to_next(int ue_index)
 	}
     else if (STAGE1_AIA_FAIL == ue_entry->ue_state)
     {
-        log_msg(LOG_ERROR, "Error AIA from HSS. Release UE session \n");
+        log_msg(LOG_ERROR, "Error AIA from HSS. Send Attach Reject & Release UE session \n");
```

Comment:
Print IMSI. We should slowly target the log cleanup which helps in debugging issues without restarting MME pod